src/Symfony/Component/Translation/Writer/TranslationWriterInterface.php

```diff
@@ -31,4 +31,16 @@
      * @throws InvalidArgumentException
      */
     public function write(MessageCatalogue $catalogue, $format, $options = array());
+
+    /**
+     * Disables dumper backup.
+     */
+    public function disableBackup();
```

Comment:
doesn't have to be interfaced:
https://github.com/symfony/symfony/blob/95dfa9343cb2ff7af26a823eb318d8f525a6708d/src/Symfony/Component/Translation/Writer/TranslationWriter.php#L42